src/hb-ot-layout.cc

```diff
@@ -1238,6 +1238,25 @@ hb_ot_layout_closure_lookups (hb_face_t      *face,
   hb_set_subtract (lookup_indexes, &inactive_lookups);
 }
 
+/**
+ * hb_ot_layout_closure_features:
+ * @face: #hb_face_t to work upon
+ * @table_tag: HB_OT_TAG_GSUB or HB_OT_TAG_GPOS
+ * @lookup_indexes: (in): collected active lookup_indices
+ * @feature_indexes: (out): all active feature indexes collected
+ *
+ * Returns all active feature indexes
+ **/
```

Comment:
Please add the new API here to `harfbuzz-sections.txt` and make this exactly iike
```
 * @feature_indexes: (out): all active feature indexes collected
 *
 * Returns all active feature indexes
 *
 * Since: REPLACEME
 **/
```
And don't replace REPLACEME tag itself so we can replace it while release. Thank you so much! :)